src/common/grpc/Cargo.toml

```
@@ -22,6 +22,7 @@ datafusion.workspace = true
 datatypes = { path = "../../datatypes" }
 flatbuffers = "23.1"
 futures = "0.3"
+lazy_static = "1.4"
```

Nit: better to write like this?
```suggestion
lazy_static.workspace = true
```